### ISSUE TITLE
Watson 1.8.0 refuses to start.

### ISSUE THREAD

**Issue by aero31aero** [TailorDev/Watson#338]
On installing from pip, I get the following traceback on invoking Watson, regardless of the arguments provided to the cli:

```
Traceback (most recent call last):
  File "/usr/local/bin/watson", line 6, in <module>
    from watson.__main__ import cli
  File "/usr/local/lib/python3.6/dist-packages/watson/__main__.py", line 2, in <module>
    from watson import cli
  File "/usr/local/lib/python3.6/dist-packages/watson/cli.py", line 473, in <module>
    pager, aggregated=False):
  File "/usr/lib/python3/dist-packages/click/decorators.py", line 170, in decorator
    _param_memo(f, OptionClass(param_decls, **attrs))
  File "/usr/local/lib/python3.6/dist-packages/watson/cli.py", line 39, in __init__
    super(MutuallyExclusiveOption, self).__init__(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/click/core.py", line 1460, in __init__
    Parameter.__init__(self, param_decls, type=type, **attrs)
TypeError: __init__() got an unexpected keyword argument 'hidden'

```

Output of pip show:

```
Name: td-watson
Version: 1.8.0
Summary: A wonderful CLI to track your time!
Home-page: https://github.com/TailorDev/Watson
Author: TailorDev
Author-email: contact@tailordev.fr
License: MIT
Location: /usr/local/lib/python3.6/dist-packages
Requires: click, requests, arrow
Required-by: 
```

Is this a known issue?

**EDIT:** I just tried the development version and it seems that it might be a python2 vs python3 thing. Keeping this open just in case there's more to this.

**Comment by jmaupetit** [comment]
@aero31aero can you confirm that @ericguirbal's work in #339 fixes this issue?

**Comment by aero31aero** [comment]
Hey, sorry for the delay, I got caught up in other work. This fixes the issue here, I tried with python 3.6.8 in a virtualenv. Thanks a ton!

**Comment by ashwani99** [comment]
This error still persists. Tried installing from `pip` and Python 3.6

Is it updated on pypa?

**Comment by jmaupetit** [comment]
This has been fixed in #339 but not released yet!

**Comment by jmaupetit** [comment]
And it has been released! Check this out!

http://tailordev.github.io/Watson/about/release-notes/#190-2020-05-27
